src/main/java/fr/xephi/authme/datasource/SQLite.java

```diff
@@ -467,6 +467,48 @@ public void setUnlogged(String user) {
     }
 
     @Override
+    public boolean hasSession(String user) {
+        String sql = "SELECT * FROM " + tableName + " WHERE LOWER(" + col.NAME + ")=?;";
```

Comment:
`SELECT col.HAS_SESSION` would be better as it retrieves only what we want from the DB.

Reply:
Yeah, i left the previous code, i'll fix that asap